.github/workflows/roll_chromium_build.yml

```diff
@@ -0,0 +1,46 @@
+name: Trigger Chromium build
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 10 * * 1,5"
+jobs:
+  trigger-chromium-build:
+    name: Trigger Build
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/github-script@v4
+        id: bump-chromium
+        with:
+          script: |
+            const fs = require('fs');
+            const response = await github.request('https://omahaproxy.appspot.com/all.json');
+            const linux = response.data.find(({os}) => os === 'linux');
+            const linuxDev = linux.versions.find(version => version.channel === 'dev');
```

Comment:
let's do canary instead? dev only updates weekly, and you have this job running bi-weekly